content/_data/changelogs/weekly.yml

```diff
@@ -7562,6 +7562,78 @@
 
   # pull: 4708 (PR title: Fix up Protector class)
 
+- version: '2.237'
+  date: 2020-05-18
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4712
+    issue: 61920
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent telemetry warnings about missing <code>javax.annotation</code> classes when running with Java 11 (regression in 2.231).
+  - type: bug
+    category: bug
+    pull: 4718
+    issue: 62231
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent Old Data Monitor from failing plugin loading in the case of class field unmarshalling issues.
+  - type: bug
+    category: bug
+    pull: 4711
+    issue: 60118
+    authors:
+    - oleg-nenashev
+    message: |-
+      Ensure that <code>UserLanguages</code> telemetry initializer always runs after extensions are augmented.
+  - type: bug
+    category: bug
+    pull: 4684
+    issue: 61956
+    authors:
+    - calvinpark
+    message: |-
+      Ensure that job/folder creation routines properly check the requested name for invalid characters.
```

Comment:
This is a `Developer:` issue since the API is not exposed to users.

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/3556